examples/asio-grpc/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-asio-grpc)
+
+# DOCUMENTATION_START {
+hunter_add_package(asio-grpc)
+find_package(asio-grpc CONFIG REQUIRED)
+
+add_executable(boo boo.cpp)
+target_link_libraries(boo PUBLIC asio-grpc::asio-grpc)
+# DOCUMENTATION_END }
```

Comment:
In other examples we additionally do a second `find_package()` to make sure the imported library target creation is well behaved. And please add a test if the license of the package is well defined

See https://github.com/cpp-pm/hunter/blob/master/examples/Eigen/CMakeLists.txt#L20 for pointers how to do that

Reply:
See https://hunter.readthedocs.io/en/latest/user-guides/cmake-user/using-license-files.html?highlight=HUNTER_INSTALL_LICENSE_FILES#using-license-files for documentation about license files and how hunter finds them